contracts/governance/Governor.sol

```diff
@@ -327,6 +327,20 @@ abstract contract Governor is Context, ERC165, EIP712, IGovernor, IERC721Receive
         return proposalId;
     }
 
+    /**
+     * @dev See {IGovernorTimelock-queue}.
+     *
+     * Note: This is an empty declaration for timelock modules to hook into.
+     */
+    function queue(
```

Comment:
This changes the interface so we may need to update `supportsInterface`. I think we agreed not to do so but flagging for visibility in case we want to add a comment about it.

Reply:
It does not. The interface ids are derived from the interface (type(IGovernor).interfaceId), which this PR does not modify.